doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -140,6 +140,7 @@ Wi-Fi
   * The :kconfig:option:`CONFIG_NRF_WIFI_IF_AUTO_START` Kconfig option to enable an application to set/unset AUTO_START on an interface.
     This can be done by using the ``NET_IF_NO_AUTO_START`` flag.
   * Support for sending TWT sleep/wake events to applications.
+  * When the QSPI peripheral is idle, set the HFCLK192M clock divider to the default /4 setting for lower power consumption.
```

Comment:
```suggestion
  * The HFCLK192M clock divider can be set to the default /4 setting for lower power consumption when the QSPI peripheral is idle.
```

Is the "/4" the value that must be entered/typed in to set the setting? If so, please add double back-ticks: ``/4``.

Reply:
In the nRF5340 PS, HFCLK192M clock divider has 3 setting values:
- Div1 --> divide HFCLK192M by 1 --> 192MHz
- Div2 --> divide HFCLK192M by 2 --> 96MHz
- Div4 --> divide HFCLK192M by 4 --> 48MHz

The current nRF7002 driver sets the HFCLK192M to Div1. It works. Nevertheless, the idle power consumption is higher than the default setting Div4.

This pull request sets the clock divider to Div2 during QSPI transactions. The clock divider is set to Div4 when the QSPI peripheral is idle.

Is the following statement okay? Is ``/4`` preferred? How about divide-by-4?

* The nRF5340 HFCLK192M clock divider is set to the default value ``Div4`` for lower power consumption when the QSPI peripheral is idle.

Reply:
Ah okay, thanks for the information.
And yes, what you have suggested with ``Div4`` is good.